godo.go

```diff
@@ -92,6 +94,22 @@ type Client struct {
 
 	// Optional rate limiter to ensure QoS.
 	rateLimiter *rate.Limiter
+
+	// Optional retry values. Setting the RetryConfig.RetryMax value enables automatically retrying requests
+	// that fail with 429 or 500-level response codes using the go-retryablehttp client
+	RetryConfig RetryConfig
+}
+
+// RetryConfig sets the values used for enabling retries and backoffs for
+// requests that fail with 429 or 500-level response codes using the go-retryablehttp client.
+// RetryConfig.RetryMax must be configured to enable this behavior. RetryConfig.RetryWaitMin and
+// RetryConfig.RetryWaitMax are optional, with the default values being 1.0 and 30.0, respectively.
+// Note: Opting to use the go-retryablehttp client will overwrite any custom HTTP client passed into New().
```

Comment:
Two suggestions – I would add an extra newline here to separate this from the rest of the GoDoc string, and, because the config takes those two pointer-to-floats, it might be worth recommending in the doc string that users use godo.PtrTo to set that.
```suggestion
//
// Note: Opting to use the go-retryablehttp client will overwrite any custom HTTP client passed into New().
//
// You can use
//  godo.PtrTo(1.0)
// to set the RetryWaitMin and RetryWaitMax values.
```